mito-ai/src/Extensions/AiChat/ChatTaskpane.tsx

```diff
@@ -195,15 +192,8 @@ const ChatTaskpane: React.FC<IChatTaskpaneProps> = ({
         agentModeEnabledRef.current = agentModeEnabled;
     }, [agentModeEnabled]);
 
-    /* 
-        Auto-scroll follow mode: tracks whether we should automatically scroll to bottom
-        when new messages arrive. Set to false when user manually scrolls up.
-    */
-    const [autoScrollFollowMode, setAutoScrollFollowMode] = useState<boolean>(true);
-    const autoScrollFollowModeRef = useRef<boolean>(autoScrollFollowMode);
-    useEffect(() => {
-        autoScrollFollowModeRef.current = autoScrollFollowMode;
-    }, [autoScrollFollowMode]);
+    // Chat scroll management
+    const { chatMessagesRef, setAutoScrollFollowMode } = useChatScroll(chatHistoryManager);
```

Comment:
Thoughts on renaming this variable from `chatMessageRef` to `chatTaskapenMessageRef` I was confused at first and thought that it was specifically referring to chat mode.